packages/sdks/web-component/test/descope-wc.test.ts

```diff
@@ -334,7 +334,10 @@ describe('web-component', () => {
     await waitFor(() => screen.getByShadowText('It works!'), {
       timeout: WAIT_TIMEOUT,
     });
-    expect(autoFocusSpy).toBeCalledWith(expect.any(HTMLElement), true, true);
+
+    await waitFor(() =>
```

Comment:
Why this test was changed? 
Can we keep the current behavior?

Reply:
auto-focus is not async, as it needs to wait for external inputs, so we have to `waitFor` it in the test.